src/components/TextInputWithCurrencySymbol/types.ts

```diff
@@ -0,0 +1,30 @@
+import type {NativeSyntheticEvent, TextInputSelectionChangeEventData} from 'react-native';
+import type {TextSelection} from '@components/Composer/types';
+
+type TextInputWithCurrencySymbolProps = {
+    /** Formatted amount in local currency  */
+    formattedAmount: string;
+
+    /** Function to call when amount in text input is changed */
+    onChangeAmount?: (value: string) => void;
+
+    /** Function to call when currency button is pressed */
+    onCurrencyButtonPress?: () => void;
+
+    /** Placeholder value for amount text input */
+    placeholder: string;
+
+    /** Currency code of user's selected currency */
+    selectedCurrencyCode: string;
+
+    /** Selection Object */
+    selection: TextSelection;
```

Comment:
```suggestion
    selection?: TextSelection;
```

According to proptypes